src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -1,7 +1,8 @@
 ﻿<ResourceDictionary

```

Comment:
Do we need to make a work item to go through all of our controls and make sure they're using these?

Reply:
Yeah I think we should, but I'm a little hesitant right now to include them in all our pages just because I'm not sure what Styles it'll override and it might mess up the UI. I think as we work page by page we can see if the Styles in `CommonResources` apply to the page itself and if it does, include them